api/kyverno/v1/rule_types.go

```diff
@@ -10,8 +10,58 @@ import (
 	apiextv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/util/validation/field"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type PolicyExceptionRequestSpec struct {
+	// PolicyName is a label to identify the target policy.
+	PolicyName string `json:"policyName" yaml:"policyName"`
+  // RuleName is a label to identify the target rule. 
+	RuleName string `json:"ruleName" yaml:"ruleName"`
+	// Reason explains why the exception exists.
+	Reason string `json:"reason" yaml:"reason"`
+	// ExcludeResources defines when this policy rule should not be applied. The exclude
+	// criteria can include resource information (e.g. kind, name, namespace, labels)
+	// and admission review request information like the name or role.
+	ExcludeResources MatchResources `json:"exclude" yaml:"exclude"`
```

Comment:
See that `MatchResources` has a field of `UserInfo`, I'm not quite familiar with that. So wondering what is that for, and would that be necessary in our scenario?

Reply:
`UserInfo` contains information on the person or app making the API call i.e. user name, groups, roles, etc. This allows writing policies rules that match or exclude requests from certain users or roles e.g. `cluster-admin`.

Reply:
That make sense. So that we should keep this field right? Thx for letting me know!